### DISCUSSION TITLE
Actividad Docker - GRUPO 13

### DISCUSSION THREAD

**Discussion by VirBruno** [frlp-utn-ingsoft/catedra#117] [Show and tell]
https://github.com/facularrauri/vetsoft

**Comment by rodriguezemautn** [comment]
Correcciones:
-el archivo Dockerfile se escribe con inicial mayuscula
-el archivo env-example no lleva . Delante
- las variables declaradas en el .env no se consumen
- Como se completa el .env?
- no estan claras las sugerencias en el README… por que usar docker run y docker-compose juntos?

Puntos aprobados:
* construye la imagen
* muy buen tamaño de imagen
* despliega el contenedor
* se ejecuta en segundo plano (-d)

Salida de comandos:
$ docker images -a
vetsoft-app   1.0.0     aaa391a6a5ee   About a minute ago   297MB
e$ docker ps -a
758b1687867f   vetsoft-app:1.0.0   "sh -c 'python manag…"   42 seconds ago   Up 40 seconds   0.0.0.0:8001->8000/tcp, :::8001->8000/tcp   vetsoft
